core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerProxy.scala

```diff
@@ -146,7 +146,7 @@ case class PreWarmedData(override val container: Container,
                          kind: String,
                          override val memoryLimit: ByteSize,
                          override val activeActivationCount: Int = 0)
-    extends ContainerStarted(container, Instant.EPOCH, memoryLimit, activeActivationCount)
+    extends ContainerStarted(container, Instant.now(), memoryLimit, activeActivationCount)
```

Comment:
overloading the lastUse property is confusing - I would move that property to ContainerInUse trait, and create a dedicated property in PrewarmedData for tracking expiration Instant or Deadline, so that every scheduled period can easily check (without calculation of expiration) whether the expiration is in the past.

Reply:
Maybe your method（I would move that property to ContainerInUse trait, and create a dedicated property in PrewarmedData for tracking expiration Instant or Deadline） is good,

But when i wrote using your method, it seems i cannot wrote it well, can you check it in your local?

Reply:
I think it takes refactoring some or most of the uses of "ContainerData" generic type. For now you can just:
* change prewarmedPool to type `var prewarmedPool = immutable.Map.empty[ActorRef, PreWarmedData]`
* add a `val created = Instant.now()` to `PrewarmedData` type
Then least the `ContainerData.lastUsed` property for use when container is actually allocated to an action.

Reply:
Updated accordingly